vendor.conf

```diff
@@ -1,3 +1,5 @@
+github.com/containers/image
```

Comment:
One-minute look, will review in detail a later:
- ^^ This seems unexpected.
- Overall having a `context.Context` in most places is definitely valuable.
- I’m not sure about passing `types.SystemContext` and `context.Context` by a single parameter, notably because `SystemContext` currently intentionally violates the “Do not store `Context`s inside a struct type” rule. Combining them may still be the right thing to do, I’ll need to take a deeper look.

Reply:
Just replying to the first point; the first line of the trash vendor.conf identifies the package root ([e.g.](https://github.com/rancher/trash/blob/master/vendor.conf#L2)); trash gives me a warning without it (presumably because I use a multiple-GOPATH and it can't identify the canonical parent on its own).

Reply:
Ah, I had no idea that was a thing. Thanks.